tests/legacy/ui/test_create_import_publish_module_and_collection.py

```diff
@@ -170,3 +170,22 @@ def test_publish_collection(self, archive_base_url, legacy_base_url,
                                         legacy_id=content_published.id).open()
         assert archive_content.title == 'CNX Automation Test Collection'
         snapshot.assert_dict_match(archive_content.stable_dict, 'legacy/col_with_m46922_1.13.json')
+
+    @markers.legacy
+    @markers.test_case('C175153')
+    @markers.slow
+    def test_empty_collection_publish_not_allowed(self, legacy_base_url, legacy_username,
+                                                  legacy_password, selenium):
+        # GIVEN a logged in user on their dashboard
+        login_page = LoginForm(selenium, legacy_base_url).open()
+        my_cnx = login_page.login(legacy_username, legacy_password)
+
+        # WHEN the user clicks to create a new collection,
+        # agrees to the license and fills in the Title
+        cc_license = my_cnx.create_collection()
+        metadata_edit = cc_license.agree().submit()
+        collection_edit = metadata_edit.fill_in_title('CNX Automation Test Collection').submit()
+
+        # THEN click publish and publish is blocked (block msg, no submit button)
+        content_published = collection_edit.publish()
+        assert content_published.block_msg
```

Comment:
I would say either assert that the block_msg is displayed (make a method that uses `is_element_displayed()`) or assert that it contains some text.